commands.md

```diff
@@ -9,6 +9,12 @@ npm run devServer -- --ip localhost --port 8443 --id 2 --targetIp localhost --ta
 npm run devServer -- --ip localhost --port 8445 --id 3 --targetIp localhost --targetPort 8440 --targetId 0
 npm run devServer -- --ip localhost --port 8444 --id 7 --targetIp localhost --targetPort 8440 --targetId 0
 
+<!-- Alvaro: npm run devClient worked for me, but it does not ends so I can't run the following command -->
+
+node client insert --ip localhost --port 8440 --id 2
```

Comment:
These are helpful! Thanks! Perhaps insert should include some other User attributes to know the synax for that. i.e. node client edit --ip localhost --port 8440 --id 2 --displayName "Pirate Bob"

Reply:
Added!